content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -34,40 +36,38 @@ there is the `VolumeSnapshotClass` resource.
 
 Users need to be aware of the following when using this feature:
```

Comment:
The previous 3 paragraphs may need to be reworded to fit the new model, especially the 3rd paragraph "While `VolumeSnapshots` allow a user to consume abstract storage resources..."

Reply:
Reworded.